diesel_codegen/src/associations/belongs_to.rs

```diff
@@ -19,10 +15,10 @@ pub fn expand_belongs_to(
 ) {
     let options = parse_association_options("belongs_to", cx, span, meta_item, annotatable);
     if let Some((model, options)) = options {
-        let association_name = options.name.name.as_str();
+        let parent_struct = options.name;
         let struct_name = model.name;
 
-        let foreign_key_name = to_foreign_key(&association_name);
+        let foreign_key_name = to_foreign_key(&parent_struct.name.as_str());
```

Comment:
`as_str()` returns an `InternedString` so it needs the additional deref to get to `&str`